components/AvalancheForecastZoneMap.tsx

```diff
@@ -230,7 +234,7 @@ const AvalancheForecastZoneCards: React.FunctionComponent<{
 
   const panResponder = useRef(
     PanResponder.create({
-      onMoveShouldSetPanResponder: () => true,
+      onMoveShouldSetPanResponder: (_event, {dx, dy}) => dx > 0 || dy > 0,
```

Comment:
this fix courtesy of https://stackoverflow.com/questions/68377590/react-native-panresponder-onrelease-event-stealing-touchable-opacity-onpress-eve